gov_uk_dashboards/components/plotly/download_button.py

```diff
@@ -1,23 +1,30 @@
 """download_button"""
-from typing import Optional
 from dash import html
 
 
-def download_button(button_text: str, button_id: Optional[str] = "download-button"):
+def download_button(button_text: str, button_id: str = "download-button"):
```

Comment:
Bearing in mind we use this function in other dashboards, I think it's best to keep the button_id as able to accept an alternative but default to "download-button" so we don't break other dashboards (including oflog who still use some components from this package).

Reply:
Also for if we were to have multiple download buttons on one page - they'd each need a unique id

Reply:
Decision needed re. whether to adapt current download_button function or create new function with different name. 
Considerations:

- oflog dashboard - have contacted Duncan to determine if they currently use download_button from this package
- our other dashboards - would we want this behaviour on our other dashboards

Reply:
created new function to maintain backwards compatability